### PR TITLE
Add the PWA Angular post

### PR DESCRIPTION
**Pull request by mgechev**
Original content [here](https://docs.google.com/document/d/1ArL5clVQd7fliN_1I_Xr6c4RmLd7dXNz2BO1kF7yt5I/edit?usp=sharing).

Hero image [here](https://pixabay.com/photos/network-cables-rj45-patch-499792/).

### CODE REVIEW

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+description: |
+  Learn how to create installable progressive Angular applications.
+authors:
+  - mgechev
+tags:
+  - angular
+  - pwa
+---
+
+In this post, you'll learn how to use the Angular command line interface (CLI) to make a [Progressive Web App](https://web.dev/discover-installable/) (PWA).
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
In this post, you'll learn how to use the Angular command line interface (CLI) to make a [Progressive Web App (PWA)](/discover-installable).
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+{% Aside %}
+
+This post assumes you're already familiar with PWAs and their benefits. If you need a refresher, check out the [Installable](https://web.dev/installable) collection.
+
+{% endAside %}
+
+
+## Create an installable PWA
+
+To make your Angular application a PWA, all you need to do is run a single command!
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
To make your Angular application a PWA, all you need to do is run a single command:
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+1. Run this command in the CLI:
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
```
Sorry, I didn't catch in the doc that this kind of just repeats the previous sentence. Doesn't really seem like it's adding anything.

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+This post assumes you're already familiar with PWAs and their benefits. If you need a refresher, check out the [Installable](https://web.dev/installable) collection.
+
+{% endAside %}
+
+
+## Create an installable PWA
+
+To make your Angular application a PWA, all you need to do is run a single command!
+
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+1. Run this command in the CLI:
+
+```bash
+ng add @angular/pwa
+```
+
+This command will:
+
+1. Create a [service worker](https://docs.google.com/document/d/1wj0NWqTmeLydh-EU3kH9orpVzXifQX6f4pjrsWo7Bv4/edit) with a default caching configuration.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
1. Create a [service worker](/precaching-with-the-angular-service-worker) with a default caching configuration.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+2. Create a [manifest file](https://web.dev/add-manifest), which tells the browser how your app should behave when installed on the user's device.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
2. Create a [manifest file](/add-manifest), which tells the browser how your app should behave when installed on the user's device.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  - pwa
+---
+
+In this post, you'll learn how to use the Angular command line interface (CLI) to make a [Progressive Web App](https://web.dev/discover-installable/) (PWA).
+
+_You can find the code sample from this guide [on GitHub](https://github.com/mgechev/manifest-web-dev)._
+
+{% Aside %}
+
+This post assumes you're already familiar with PWAs and their benefits. If you need a refresher, check out the [Installable](https://web.dev/installable) collection.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
This post assumes you're already familiar with PWAs and their benefits. If you need a refresher, check out the [Installable](/installable) collection.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+```bash
+ng add @angular/pwa
+```
+
+This command will:
+
+1. Create a [service worker](https://docs.google.com/document/d/1wj0NWqTmeLydh-EU3kH9orpVzXifQX6f4pjrsWo7Bv4/edit) with a default caching configuration.
+2. Create a [manifest file](https://web.dev/add-manifest), which tells the browser how your app should behave when installed on the user's device.
+3. Adds a link to the manifest file in `index.html`.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
3. Add a link to the manifest file in `index.html`.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+4. Adds the [`theme-color`](https://web.dev/themed-omnibox) `<meta>` tag to `index.html`.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
4. Add the [`theme-color`](/themed-omnibox) `<meta>` tag to `index.html`.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+5. Creates app icons in the `src/assets` directory.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
5. Create app icons in the `src/assets` directory.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+## Customize your PWA
+
+The [Precaching with the Angular service worker](https://web.dev/precaching-with-the-angular-service-worker/) post explains how to configure the Angular service worker. There you can find how to specify which resources you want the service worker to cache and what strategy it should use to do so.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
The [Precaching with the Angular service worker](/precaching-with-the-angular-service-worker) post explains how to configure the Angular service worker. There you can find how to specify which resources you want the service worker to cache and what strategy it should use to do so.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+
+The manifest file of your app lets you specify your app's name, short name, icons, theme color, and other details. Read about the full set of properties you can set on the MDN "[Web App Manifest](https://web.dev/add-manifest/)" page.
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
The manifest file of your app lets you specify your app's name, short name, icons, theme color, and other details. Read about the full set of properties you can set on the [Add a web app manifest](/add-manifest/) post.
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ]
+}
+```
+
+You can customize any of these properties by changing the relevant value in `manifest.webmanifest`.
+
+A PWA references its manifest file with a `link` element in `index.html`. Once the browser find" the reference it'll show the "Add to Home Screen" prompt:
```
Comment (mfriesenhahn) [inline src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]:
```suggestion
A PWA references its manifest file with a `link` element in `index.html`. Once the browser finds the reference, it'll show the **Add to Home screen** prompt:
```

File: src/site/content/en/angular/creating-pwa-with-angular-cli/index.md (hunk 5 above)
Comment (mgechev) [reply src/site/content/en/angular/creating-pwa-with-angular-cli/index.md]: Ups, good catch!

**Comment by mgechev** [comment]
@mfriesenhahn thanks for all the fixes!

**Comment by mfriesenhahn** [comment]
Boom! Merged! 🎉

**Pull request merged**